server/apiRoutes/group.js

```diff
@@ -225,6 +225,17 @@ router.delete('/removeself', async (req, res, next) => {
   }
 });
 
+//get all events for a specific group
+router.get('/:id/events', async (req, res, next) => {
+  try {
+    const group = req.params.id;
```

Comment:
I think you need to query for the group here?

Reply:
fixed